### ISSUE TITLE
Can loginkit be used without gdm

### ISSUE THREAD

**Issue by juanitotc** [dimkr/LoginKit#11]
I have gnome-3.10.x compiled from source working by starting gnome-session directly with "startx" - i.e. without using gdm on a distro that does not use systemd (tinycorelinux).

I am unable to get gnome-3.14.x or gnome-3.16.x working by starting gnome-session directly with "startx". The gnome-session debug output it not very useful, but it does give:

`(gnome-settings-daemon:20943): media-keys-plugin-WARNING **: Unable to inhibit keypresses: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.login1 was not provided by any .service files`

[note that gnome-3.10.x also gives this warning, but works anyway]

and then:

`gnome-session2[20901]: WARNING: Application 'gnome-shell.desktop' failed to register before timeout`

Is it possible to use loginkit directly with gnome-session? If so, how and where should it be started?

**Comment by dimkr** [comment]
Nope, it won't work. I know for sure that gnome-shell depends on logind pretty heavily, LoginKit hasn't reached that stage yet. The focus has been getting GDM to work, but I suspended LoginKit's development, at least until Devuan 2.

On Wed, 13 May 2015 03:48:20 -0700
juanitotc notifications@github.com wrote:

> I have gnome-3.10.x compiled from source working by starting gnome-session directly with "startx" - i.e. without using gdm on a distro that does not use systemd (tinycorelinux).
> 
> I am unable to get gnome-3.14.x or gnome-3.16.x working by starting gnome-session directly with "startx". The gnome-session debug output it not very useful, but it does give:
> 
> `(gnome-settings-daemon:20943): media-keys-plugin-WARNING **: Unable to inhibit keypresses: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name org.freedesktop.login1 was not provided by any .service files`
> 
> [note that gnome-3.10.x also gives this warning, but works anyway]
> 
> and then:
> 
> `gnome-session2[20901]: WARNING: Application 'gnome-shell.desktop' failed to register before timeout`
> 
> Is it possible to use loginkit directly with gnome-session? If so, how and where should it be started?
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/dimkr/LoginKit/issues/11

## 

Dima Krasner, dimakrasner.com
